.travis.yml

```
@@ -1,18 +1,24 @@
 language: php
 
+dist: trusty
+
 sudo: false
 
-jobs:
+matrix:
+  fast_finish: true
   include:
-    - { php: 5.3, dist: precise }
-    - { php: 5.4, dist: precise }
-    - { php: 5.5, dist: precise }
+    - php: 5.3
+      dist: precise
+    - php: 5.4
+    - php: 5.5
     - php: 5.6
+    - php: 7.0
+    - php: 7.1
+    - php: 7.2
+    - php: 7.3
+    - php: 7.4
```

```suggestion
    - php: 7.4
      dist: bionic
```

That should enable PHP 7.4 as per @kuraobi's comment

---

Yep, already tested here https://github.com/Behat/Transliterator/pull/30 , everything green